frontend/app/src/entities/diff/domain/resolve-conflict.mutation.ts

```diff
@@ -0,0 +1,15 @@
+import { useMutation } from "@tanstack/react-query";
+
+import {
+  type ResolveConflictParams,
+  resolveConflict,
+} from "@/entities/diff/domain/resolve-conflict";
+
+export const RESOLVE_CONFLICT_KEY = "update-diff";
```

Comment:
wrong key variable. Also `mutationKey` is optional, is you do not need to grab its state, you can omit it